lib/services/build.ts

```diff
@@ -228,6 +228,12 @@ export class BuildService implements Project.IBuildService {
 				return buildResult;
 			} else if(settings.platform === "WP8") {
 				var buildCompanyHubApp = !settings.downloadFiles;
+				if(this.$project.projectData.WPSdk === "8.1" && ((options.release && settings.downloadFiles) || settings.buildForTAM)) {
```

Comment:
These properties should be valid in any build :smiley_cat: In our semantic the only difference between release and debug is the livesync plugin. And if they enter invalid value nd try just to build they will get errors the same way they will, when "publishing"